src/vs/workbench/test/electron-browser/textsearch.perf.test.ts

```diff
@@ -114,7 +118,7 @@ suite('TextSearch performance', () => {
 		const finishedEvents = [];
 		return runSearch() // Warm-up first
 			.then(() => {
-				if (testWorkspaceArg) { // Don't measure by default
+				if (testWorkspacePath) { // Don't measure by default
```

Comment:
Btw., any unit tests that would make sense adding or are the existing ones still sufficient?